internal/controller/volsync/vshandler.go

```diff
@@ -2036,8 +2034,8 @@ func (v *VSHandler) createReadOnlyPVCFromSnapshot(rd *volsyncv1alpha1.Replicatio
 			pvc.Spec.AccessModes = accessModes
 			pvc.Spec.StorageClassName = rd.Spec.RsyncTLS.StorageClassName
 
-			// Only set when initially creating
-			pvc.Spec.DataSource = &snapshotRef
+			pvc.Spec.DataSource = snapshotRef
```

Comment:
Why snapshotRef changed? I don't see any other change that explain this.

Reply:
It changed to a pointer